tests/plots/test_bar.py

```
@@ -0,0 +1,321 @@
+        "sales_q2": [1100, 1600, 2100, 2600],
+    }
+    return pd.DataFrame(data)
+
+
+@pytest.fixture()
+def sample_series():
+    """A sample series for testing."""
+    return pd.Series([1000, 1500, 2000, 2500], index=["A", "B", "C", "D"])
+
+
+@pytest.fixture()
```

Ruff

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)